scm/driver/bitbucket/webhook.go

```diff
@@ -53,6 +53,23 @@ func (s *webhookService) Parse(req *http.Request, fn scm.SecretFunc) (scm.Webhoo
 		if hook != nil {
 			hook.(*scm.PullRequestHook).Action = scm.ActionClose
 		}
+	case "pullrequest:comment_created":
+		hook, err = s.parsePullRequestCommentHook(data)
+		if hook != nil {
+			hook.(*scm.IssueCommentHook).Action = scm.ActionCreate
+		}
+	case "pullrequest:comment_updated":
+		// Bitbucket PR Comment Update is unreliable and does not send events
+		// most of the time https://github.com/iterative/cml/issues/817
+		hook, err = s.parsePullRequestCommentHook(data)
+		if hook != nil {
+			hook.(*scm.IssueCommentHook).Action = scm.ActionUpdate
```

Comment:
@rutvijmehta-harness One small change. Comments should use ActionEdit not ActionUpdate as per https://github.com/drone/go-scm/blob/master/scm/driver/github/webhook.go#L125-L136

Reply:
Otherwise it looks 👍

Reply:
Done